rpc/ethereum/backend/backend.go

```diff
@@ -954,9 +954,34 @@ func (e *EVMBackend) ChainConfig() *params.ChainConfig {
 }
 
 // SuggestGasTipCap returns the suggested tip cap
-// always return zero since we don't support tx prioritization yet.
-func (e *EVMBackend) SuggestGasTipCap() (*big.Int, error) {
-	return big.NewInt(0), nil
+// Although we don't support tx prioritization yet, but we return a positive value to help client to
+// mitigate the base fee changes.
+func (e *EVMBackend) SuggestGasTipCap(baseFee *big.Int) (*big.Int, error) {
+	if baseFee == nil {
+		// london hardfork not enabled or feemarket not enabeld
+		return big.NewInt(0), nil
+	}
+
+	params, err := e.queryClient.FeeMarket.Params(e.ctx, &feemarkettypes.QueryParamsRequest{})
+	if err != nil {
+		return nil, err
```

Comment:
I think its detable but I believe returning nil here makes sense since it is an actual internal error when / if this happens. Returning a default value could lead to believe the endpoint is actually working. Which could be confusing